action.yml

```
@@ -0,0 +1,76 @@
+name: 'Run quickstart'
+description: 'Runs quickstart docker image'
+inputs:
+  version:
+    description: "Version (docker tag) of quickstart image to use"
+    required: true
+    default: "latest"
+  network:
+    description: "Network to run"
+    default: "local"
+  enable_logs:
+    description: "Boolean flag enabling the logs"
+    default: "true"
+  health_interval:
+    description: "Time between running the check (in seconds)"
+    default: "10"
+  health_timeout:
+    description: "Maximum time to allow one check to run (in seconds)"
+    default: "5"
+  health_retries:
+    description: "Consecutive failures needed to report unhealthy"
+    default: "50"
+runs:
+  using: "composite"
+  steps:
+    - name: "Fail on windows"
+      if: runner.os == 'Windows'
+      shell: powershell
+      run: |
+        echo "Windows runners are not supported"
+        exit 1
+    - name: "Fail on windows"
+      if: runner.os == 'macOS' && (runner.arch == 'ARM' || runner.arch == 'ARM64')
+      shell: bash
+      run: |
+        echo "ARM MacOS runner is not yet supported, see https://github.com/douglascamata/setup-docker-macos-action"
+        exit 1
+    - name: Setup docker (missing on MacOS)
+      shell: bash
+      if: runner.os == 'macos'
+      run: |
+        brew install docker
+        brew install colima
+        colima start
+    - run: >
+        docker run -d --name quickstart 
```

Suggest naming the container stellar, instead of quickstart. That's typically what the CLI names the containers.
```suggestion
        docker run -d --name stellar 
```